docs/faq.md

```diff
@@ -223,3 +223,10 @@ RUN poetry install --no-dev
 
 The two key options we are using here are `--no-root` (skips installing the project source) and `--no-directory` (skips installing any local directory path dependencies, you can omit this if you don't have any).
 [More information on the options available for `poetry install`]({{< relref "cli#install" >}}).
+
+
+### My requests are timing out!
+
+Poetry's default HTTP request time out is 15 seconds, the same as `pip`.
+Similar to `PIP_REQUESTS_TIMEOUT`, the experimental environment variable `POETRY_REQUESTS_TIMEOUT`
+can be set to alter this value. This environment variable is **unstable**.
```

Comment:
```suggestion
can be set to alter this value.
```

The previous sentence already says _"experimental environment variable"_. You could emphasize **experimental** if you like. IMO, that's sufficient. The last sentence could also be understood to mean that setting the environment variable does not work reliably.

Reply:
Done